public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -34,7 +34,7 @@ export const ReceiversTable: FC<Props> = ({ config, alertManagerName }) => {
       addButtonLabel="New contact point"
```

Comment:
I'm a bit confused by the lingio in the code. Looks like this functionality is called "ContactPoint" in the UI but we call it Receivers in the code. How come?